sdk/spring-experimental/spring-cloud-azure-native-configuration/README.md

```diff
@@ -1,4 +1,4 @@
-# Spring Cloud Azure Spring Native Support client library for Java
+# Spring Cloud Azure Native Configuration
 
 The Azure Spring Native Support client library provides support for Spring Boot applications using Spring Cloud Azure Starters to be built as [GraalVM native
```

Comment:
The Spring Cloud Azure Native Support library

Reply:
support for building Spring Boot applications using Spring Cloud Azure Starters to [GraalVM native images]